std/array_list.zig

```diff
@@ -174,4 +228,4 @@ test "insert ArrayList test" {
     const items = []const i32 { 1 };
     try list.insertSlice(0, items[0..0]);
     assert(list.items[0] == 5);
-}
+}
```

Comment:
The newlines were removed from the end of all the files you modified.

Reply:
Didn't remember having them be part of a standard, if they are I'll add them back; just my visual studio code having its own mind :).